css/properties/flex.json

```diff
@@ -0,0 +1,109 @@
+{
+  "css": {
+    "properties": {
+      "flex": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flex",
+          "support": {
+            "webview_android": {
+              "version_added": "4.4"
```

Comment:
-webkit- version has been supported and stil is: "true"

Reply:
Done.